### PR TITLE
Some Tweaks to the InstallSitePackage installer

### PR DESCRIPTION
**Pull request by bwaidelich**
* Apply cosmetic fixes from #42 (typos, unused namespace imports, ...)
* Extract installable "distributions" to static method (to make it
  easier to adjust/extend)
* Replace "empty Neos" with a bare bones installation that only contains
  the necessary packages

### CODE REVIEW

File: Build/Classes/Composer/InstallSitePackage.php [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
 use Composer\Script\Event;
 use Symfony\Component\Console\Input\ArrayInput;
 use Symfony\Component\Console\Output\ConsoleOutput;
-use Neos\Splash\DistributionBuilder\Service\PackageService;
-use Neos\Splash\DistributionBuilder\Service\JsonFileService;
-use Neos\Splash\DistributionBuilder\Domain\ValueObjects\PackageRequirement;
 
 /**
- * A comopser post-create-project script to allow a choice of adding the demo site (and possible other things in the future)
+ * A composer post-create-project script to allow a choice of adding the demo site (and possible other things in the future)
```
Comment (albe) [inline Build/Classes/Composer/InstallSitePackage.php:11]: I think `comopser` is an infinitely cuter name, but sure, better let's be correct than cute here ;)

**Review by albe — APPROVED** [review]

File: Build/Classes/Composer/InstallSitePackage.php (hunk 1 above)
Comment (bwaidelich) [reply Build/Classes/Composer/InstallSitePackage.php:11]: I had to laugh about this one, too :)

**Comment by bwaidelich** [comment]
*ping*

**Comment by bwaidelich** [comment]
*ping²* (also see https://github.com/neos/neos-base-distribution/pull/42 and https://discuss.neos.io/t/rfc-post-create-project-tasks/5141/14)

**Comment by bwaidelich** [comment]
I'm giving up :)
Closing to remove this from my list of pending PRs – feel free to reopen

**Pull request closed without merging**